copilot-chat-request.el

```diff
@@ -296,14 +295,14 @@ Argument RESPONSE is request-response object."
   "Enable policy for MODEL-ID."
   (let ((url (format "https://api.githubcopilot.com/models/%s/policy" model-id))
         (headers (copilot-chat--get-headers))
-        (data (json-encode '((state . "enabled")))))
+        (data (json-serialize '((state . "enabled")))))
     (when copilot-chat-debug
       (message "Enabling policy for model %s" model-id))
     (request url
       :type "POST"
       :headers headers
       :data data
-      :parser 'json-read)))
+      :parser (apply-partially 'json-parse-buffer :object-type 'alist :false-object :json-false :null-object nil))))
```

Comment:
`json.c` does not use `:json-false`, but `:false` is assigned by default, so there is no need to force a custom setting.